src/System.Net.Security/src/System/Net/Unix/CertModule.cs

```diff
@@ -20,7 +26,23 @@ internal override SslPolicyErrors VerifyCertificateProperties(X509Chain chain, X
                 sslPolicyErrors |= SslPolicyErrors.RemoteCertificateChainErrors;
```

Comment:
This is actually related to https://github.com/dotnet/corefx/pull/3432. Windows code throws a CryptoException if Build fails. Unix code should behave the same way